jupyter_book/commands/__init__.py

```diff
@@ -172,7 +172,7 @@ def build(path_book, path_output, config, toc, warningiserror, builder):
                 with cd(OUTPUT_PATH):
                     output = subprocess.run([makecmd, "all-pdf"])
                     if output.returncode != 0:
-                        return output.returncode
+                        exit(output.returncode)
```

Comment:
this should be `return` and we need to check if `circle` is acting on status codes. Does the `return` get returned by jupyter_book?

Reply:
Yes, it did get returned by `jupyter_book` in my local system. But, did not seem to work in `circleci`

Reply:
why not just check the output code yourself in the python and raise an `_error(` depending on its value?

Reply:
ooh ok. so you are basically saying to use `_error` instead of `exit`.